website-generator/resources/src/table_search.js

```diff
@@ -4,18 +4,28 @@
 
 // Table search
 (function () {
-  function onSearch (inputId, tableId) {
+  function onSearch(inputId, tableId, tableType) {
```

Comment:
Please add some documentation to this function. Its signature is getting complicated. Let's write in the documentation string what expected values for each are.